x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/confusion_matrix_help_popover.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  HelpPopover,
+  HelpPopoverButton,
+} from '../../../../../components/help_popover/help_popover';
+
+export const MulticlassConfusionMatrixHelpPopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  return (
+    <HelpPopover
+      anchorPosition="upCenter"
+      button={
+        <HelpPopoverButton
+          onClick={() => {
+            setIsPopoverOpen(!isPopoverOpen);
+          }}
+        />
+      }
+      closePopover={() => setIsPopoverOpen(false)}
+      isOpen={isPopoverOpen}
+      title={i18n.translate('xpack.ml.dataframe.analytics.confusionMatrixPopoverTitle', {
+        defaultMessage: 'Multiclass confusion matrix',
```

Comment:
I wonder if we should synch with the label for this section, for example:
```suggestion
        defaultMessage: 'Normalized confusion matrix',
```
Otherwise, I start pondering what the difference is between "normalized" and "multiclass" and whether the information is applicable to what I'm seeing or if only certain matrices are multiclass (down the rabbit hole I go!)